user_guide_src/source/installation/upgrade_database.rst

```diff
@@ -0,0 +1,67 @@
+Upgrade Database
+################
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations
+==============
+
+- `Database Reference Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/database/index.html>`_
+- :doc:`Working with Databases Documentation Codeigniter 4.X </database/index>`
+
+
+What has been changed
+=====================
+- The functionality in CI3 is basically the same as in CI4.
+- The method names have changed to camelCase and the query builder now needs to be initialized before you can run queries on it.
+
+Upgrade Guide
+=============
+1. Add your database credentials to ``app/Config/Database.php``. The options are pretty much the same as in CI3 only some names have changed slightly.
+2. Everywhere you have used the database you have to replace ``$this->load->database();`` with ``$db = \Config\Database::connect();``.
```

Comment:
Lets use `db_connect()` instead of `\Config\Database::connect()`. It's a little nicer syntax to use and simpler for people to remember.

Reply:
```suggestion
2. Everywhere you have used the database you have to replace ``$this->load->database();`` with ``$db = db_connect();``.
```